### PR TITLE
Fixed formatting of examples and removed JSON-P section

### PR DESCRIPTION
**Pull request by MateoAO**
I intended just to fix the formatting of this section
image

but one thing led to another, see notes lines.

### CODE REVIEW

File: content/docs/index.md [hunk 1]
```diff
@@ -1,74 +1,65 @@
 +++
 date = "2016-02-14T16:11:58+05:30"
-title = "Strava API v3 Documentation"
+title = "Strava API V3 Documentation"
```
Comment (MateoAO) [inline content/docs/index.md]: Changed the case of v3 -> V3

File: content/docs/index.md [hunk 2]
```diff
@@ -1,74 +1,65 @@
...
 +++
 
-The Strava v3 API is a publicly available interface allowing developers access to the rich Strava dataset. The interface is stable and currently used by the Strava mobile applications.
+The Strava V3 API is a publicly available interface allowing developers access to the rich Strava dataset. The interface is stable and currently used by the Strava mobile applications.
 
 ## Access
 
-All calls to the Strava API require an `access_token` defining the athlete and application making the call. Any registered Strava user can obtain an access_token by first creating an application at labs.strava.com/developers.
+All calls to the Strava API require an `access_token` defining the athlete and application making the call. Any registered Strava user can obtain an access_token by first creating an application at https://www.strava.com/settings/api.
```
Comment (MateoAO) [inline content/docs/index.md]: Fixed link to proper location to create an API app

File: content/docs/index.md [hunk 3]
```diff
@@ -1,74 +1,65 @@
...
 
 The API application settings page provides a public access token to get started. See the [Authentication](/docs/authentication) page for more information about generating access tokens and the OAuth authorization flow.
 
 Generally speaking, a Strava API application only has access to a user’s data after the user has authorized the application to use it. Segment and segment leaderboard data is available to all applications.
 
+## Rate Limiting
```
Comment (MateoAO) [inline content/docs/index.md]: Moved this up above the fold

File: content/docs/index.md [hunk 4]
```diff
@@ -1,74 +1,65 @@
...
+
+Strava API usage is limited on a per-application basis using a short term, 15 minute, limit and a long term, daily, limit. The default rate limit allows 600 requests every 15 minutes, with up to 30,000 requests per day. This limit allows applications to make 40 requests per minute for about half the day.
+
+An application’s short term, 15 minute, limit is reset at natural 15 minute intervals corresponding to 0, 15, 30 and 45 minutes after the hour. The long term, daily limit, it resets at midnight UTC. Requests exceeding the limit will return 403 Forbidden along with a JSON error message. Note that requests violating the short term limit will still count toward the long term limit.
+
+An application’s limits and usage are reported on the API application settings page as well as returned with every API request as part of the HTTP Headers.
+
+* **X-RateLimit-Limit: integer, integer** short term followed by long term limit
+* **X-RateLimit-Usage: integer, integer** short term followed by long term usage
```
Comment (MateoAO) [inline content/docs/index.md]: image

File: content/docs/index.md [hunk 5]
```diff
@@ -1,74 +1,65 @@
...
-Resource states
-1	meta
-2	summary
-3	detailed
-Listed in increasing levels of detail.
+Resource states, in increasing levels of detail.
+
+1. meta
+2. summary
+3. detailed
```
Comment (MateoAO) [inline content/docs/index.md]: image

File: content/docs/index.md [hunk 6]
```diff
@@ -1,74 +1,65 @@
...
 
 ### Pagination
 
 Requests that return multiple items will be paginated to 30 items by default. The page parameter can be used to specify further pages or offsets. The per_page may also be used for custom page sizes up to 200. Note that in certain cases, the number of items returned in the response may be lower than the requested page size, even when that page is not the last. If you need to fully go through the full set of results, prefer iterating until an empty page is returned.
 
-### JSON-P callbacks
```
Comment (MateoAO) [inline content/docs/index.md]: Removed this

File: content/docs/index.md [hunk 7]
```diff
@@ -1,74 +1,65 @@
...
-
 ### Polylines
 
-Activity and segment API requests may include summary polylines of their respective routes. The values are string encodings of the latitude and longitude points using the Google encoded polyline algorithm format.
+Activity, segment and route API requests may include summary polylines of their respective paths. The values are string encodings of the latitude and longitude points using the [Google encoded polyline algorithm](https://developers.google.com/maps/documentation/utilities/polylinealgorithm) format.
 
 ### Dates
 
-Dates and times follow the ISO 8601 standard, unless noted. A few examples:
+Dates and times follow the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) standard, unless noted. A few examples:
```
Comment (MateoAO) [inline content/docs/index.md]: Added Link

File: content/docs/index.md [hunk 8]
```diff
@@ -1,74 +1,65 @@
...
 
-2008-04-12T00:00:00Z
-2012-12-30T17:08:56Z
-2013-08-23T15:46:20Z
+* 2015-08-23T15:46:20Z
```
Comment (MateoAO) [inline content/docs/index.md]: image

File: content/docs/index.md [hunk 9]
```diff
@@ -1,74 +1,65 @@
...
-
-An application’s limits and usage are reported on the API application settings page as well as returned with every API request as part of the HTTP Headers.
-
-X-RateLimit-Limit	integer, integer
-two comma separate values, short term followed by long term limit
-X-RateLimit-Usage	integer, integer
-two comma separate values, short term followed by long term usage
-
-As an application grows its rate limit may need to be reassessed. To request an adjustment, email [api@strava.com](mailto:api@strava.com?subject=Rate%20Limits) with the subject "Rate Limits".
+* **HEAD** can be issued against any resource to get just the HTTP header info
```
Comment (MateoAO) [inline content/docs/index.md]: image

File: content/docs/index.md [hunk 10]
```diff
@@ -1,74 +1,65 @@
...
-### JSON-P callbacks
-
-Specify the `callback=<function_name>` parameter to request a JSONP compatible response. This is compatible with JavaScript libraries such as jQuery.
-
-When developing JavaScript applications, please take care to hide the access_token, i.e. do not include it in any deep linked URLs.
-
 ### Polylines
 
-Activity and segment API requests may include summary polylines of their respective routes. The values are string encodings of the latitude and longitude points using the Google encoded polyline algorithm format.
+Activity, segment and route API requests may include summary polylines of their respective paths. The values are string encodings of the latitude and longitude points using the [Google encoded polyline algorithm](https://developers.google.com/maps/documentation/utilities/polylinealgorithm) format.
```
Comment (MateoAO) [inline content/docs/index.md]: Added link

File: content/docs/index.md [hunk 11]
```diff
@@ -1,74 +1,65 @@
...
+All calls to the Strava API require an `access_token` defining the athlete and application making the call. Any registered Strava user can obtain an access_token by first creating an application at https://www.strava.com/settings/api.
 
 The API application settings page provides a public access token to get started. See the [Authentication](/docs/authentication) page for more information about generating access tokens and the OAuth authorization flow.
 
 Generally speaking, a Strava API application only has access to a user’s data after the user has authorized the application to use it. Segment and segment leaderboard data is available to all applications.
 
+## Rate Limiting
+
+Strava API usage is limited on a per-application basis using a short term, 15 minute, limit and a long term, daily, limit. The default rate limit allows 600 requests every 15 minutes, with up to 30,000 requests per day. This limit allows applications to make 40 requests per minute for about half the day.
```
Comment (mindy-stivers) [inline content/docs/index.md]: While we're here... nit: rewrite for clarity 
> short term, 15 minute, limit

and 
> long term, daily, limit

Maybe something like... 
>a short term (15 minute) limit

or
>a 15 minute short term limit

or
>a short term limit of 15 minutes

File: content/docs/index.md [hunk 12]
```diff
@@ -1,74 +1,65 @@
...
+
+An application’s short term, 15 minute, limit is reset at natural 15 minute intervals corresponding to 0, 15, 30 and 45 minutes after the hour. The long term, daily limit, it resets at midnight UTC. Requests exceeding the limit will return 403 Forbidden along with a JSON error message. Note that requests violating the short term limit will still count toward the long term limit.
```
Comment (mindy-stivers) [inline content/docs/index.md]: Same thing here with 
>short term, 15 minute, limit

and 
>long term, daily limit,

Comment (mindy-stivers) [inline content/docs/index.md]: >long term, daily limit, it

The above quote probably should not have the word "it" in it.

**Comment by mindy-stivers** [comment]
Much easier to read! 🎉

**Pull request merged**